.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,23 @@
+name: Release Drafter
```

Comment:
do you want to check in the workflow as well as the config file?

Reply:
I removed it in the latest push.  Thanks.